ext/tidy/tests/parsing_file_too_large.phpt

```diff
@@ -0,0 +1,56 @@
+--TEST--
+Trying to parse a file that is too large (over 4GB)
+--EXTENSIONS--
+tidy
+--SKIPIF--
+<?php
+if (PHP_INT_SIZE != 8) die("skip this test is for 64bit platform only");
+if (getenv("SKIP_SLOW_TESTS")) die("skip slow test");
```

Comment:
Do we really need this test? It causes issues in CI. We should at least disable it for ASAN/UBSAN/MSAN.

Reply:
I'd prefer to have such a test, but skiping it for ASAN/UBSAN/MSAN can make sense, as this what caught by ZMM anyway.